information-schema/information-schema.md

```diff
@@ -52,6 +52,9 @@ Many `INFORMATION_SCHEMA` tables have a corresponding `SHOW` command. The benefi
 | Table Name                                                                              | Description |
 |-----------------------------------------------------------------------------------------|-------------|
 | [`ANALYZE_STATUS`](/information-schema/information-schema-analyze-status.md)            | Provides information about tasks to collect statistics. |
+| [`CLIENT_ERRORS_SUMMARY_BY_HOST`](/information-schema/client-errors-summary-by-host.md)  | Provides a summary of errors and warnings generated by clients. |
```

Comment:
I am a bit confused about how "errors and warnings generated by clients" are also returned to the clients.